diff --git a/swig/Python3/Makefile b/swig/Python3/Makefile
index 5bf8b15..db8fbb4 100644
--- a/swig/Python3/Makefile
+++ b/swig/Python3/Makefile
@@ -43,11 +43,7 @@ else
   CXXFLAGS += -std=c++0x
   # Make sure you have Atlas installed. You can statically link Atlas if you
   # would like to be able to move the library to a machine without Atlas.
-  ifneq ("$(ldconfig -p | grep lapack_atlas)","")
-    LDLIBS := -lm -ldl -lf77blas -lcblas -llapack_atlas -latlas
-  else
-    LDLIBS := -lm -ldl -lf77blas -lcblas -llapack -latlas
-  endif
+  LDLIBS := -lm -ldl -llapack -lopenblas
   SNOWBOYDETECTLIBFILE = $(TOPDIR)/lib/ubuntu64/libsnowboy-detect.a
   ifneq (,$(findstring arm,$(shell uname -m)))
     SNOWBOYDETECTLIBFILE = $(TOPDIR)/lib/rpi/libsnowboy-detect.a
